README.md

```diff
@@ -64,10 +64,11 @@ erDiagram
     flag {
         string meta_sale_document_num PK
         date rolling_window
-        bigint sv_is_outlier
-        bigint sv_is_ptax_outlier
-        bigint sv_is_heuristic_outlier
-        string sv_outlier_type
+        boolean sv_is_outlier
+        boolean sv_is_ptax_outlier
+        boolean sv_is_heuristic_outlier
+        boolean ptax_flag_original
+        boolean sv_outlier_type
```

Comment:
**nitpick:** I would put these in the order they appear in the database. In Athena they appear in a different order, but it could be because of an old crawl.

Reply:
The order of those columns are selected downstream when we subset by columns.

Reply:
Fixed